packages/rpc-provider/package.json

```diff
@@ -32,7 +32,7 @@
     "@polkadot/x-fetch": "^8.6.1",
     "@polkadot/x-global": "^8.6.1",
     "@polkadot/x-ws": "^8.6.1",
-    "@substrate/connect": "^0.6.3",
+    "@substrate/connect": "0.7.0-alpha.0",
```

Comment:
Thank you.

Reply:
Totally! It was the right thing to do! So, thank you for pointing it out!